src/test/java/seedu/findvisor/model/person/RemarkTest.java

```diff
@@ -0,0 +1,61 @@
+package seedu.findvisor.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class RemarkTest {
+    public static final String REMARK_VALUE = "Wants to afford a car after 5 years of working";
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Remark(null));
+    }
+
+    @Test
+    public void createRemark_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> Remark.createRemark(null));
+    }
+
+    @Test
+    public void createRemark_blank_returnsOptionalEmpty() {
+        // Blank Remarks are still valid, although they are impossible to create through commands.
+        assertTrue(Remark.createRemark("").isEmpty());
+        assertTrue(Remark.createRemark("  ").isEmpty());
+    }
+
+    @Test
+    public void createRemark_notBlank_returnsOptionalRemark() {
+        assertEquals(Remark.createRemark(REMARK_VALUE).map(r -> r.value).orElse(""), REMARK_VALUE);
+
+        // Leading and trailing whitespace not trimmed without ParseUtil.parseRemark
+        assertNotEquals(Remark.createRemark("  " + REMARK_VALUE).map(r -> r.value).orElse(""), REMARK_VALUE);
+    }
+
+    @Test
+    public void equals() {
+        Remark remark = new Remark(REMARK_VALUE);
+
+        // same values -> returns true
+        assertTrue(remark.equals(new Remark("Wants to afford a car after 5 years of working")));
```

Comment:
Instead of hardcoding the same string value, it would be clearer and less error prone to use the REMARK_VALUE variable.
```suggestion
        assertTrue(remark.equals(new Remark(REMARK_VALUE)));
```
Applies to line 50 too